Tests/PostgresNIOTests/PostgresNIOTests.swift

```diff
@@ -1028,6 +1028,37 @@ final class PostgresNIOTests: XCTestCase {
             XCTFail("\(error)")
         }
     }
+
+    func testPreparedQuery() throws {
+         let conn = try PostgresConnection.test(on: eventLoop).wait()
+
+         defer { try! conn.close().wait() }
+         let prepared = try conn.prepare(query: "SELECT 1 as one;").wait()
```

Comment:
Is this missing a deallocate for the prepared query? Maybe we should add an assertion to make sure it gets deallocated.

Reply:
A query doesn't need to get deallocated, because the session is closed by the end of this test.

Maybe deallocate is a bad name here because in most cases it refers to memory.